src/scrabbleScore.js

```diff
@@ -24,17 +24,38 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+
+  let pointCount = 0;
+  let letters = Object.keys(SCRABBLE_TILE_SCORES);
+  let values = Object.values(SCRABBLE_TILE_SCORES);
+
+  for (let j = 0; j <= word.length ; j++ ) {
+
+    for (let i = 0; i <= 25; i++ ) {
```

Comment:
`SCRABBLE_TILE_SCORES` is an object whose keys are letters. `word.charAt(j)` is a letter. Access the object values directly using `[]`.

If you're worried about `word` containing letters not in `SCRABBLE_TILE_SCORES` then check if the value for the given key is `undefined`.